SamLau95/nbinteract
# Show code execution error messages on HTML page

Closes #61

This pr displays error messages for widget initialization in the frontend
itself instead of hiding them all in the console.

Only error messages for widget cells get shown; other messages are still logged
to the console.

**Test plan:**

- Run `make serve -j2`
- Run `python -m http.server` in the `packages/nbinteract-core` directory
- Run `nbinteract notebooks/Test.ipynb -t local` from the project root and open
  the `notebooks/Test.html` file.
- Verify that the code error messages only appear for widget cells when widgets
  are generated:

![screenshot 2018-03-13 14 23 25](https://user-images.githubusercontent.com/2468904/37370948-017fdad2-26cb-11e8-89a2-c0fcea3715e3.png)
